public/images/favicon/manifest.json

```diff
@@ -1,18 +1,20 @@
 {
     "name": "Decred",
+    "short_name": "Decred",
     "icons": [
         {
-            "src": "/images/favicon/android-chrome-192x192.png?v=fi5jKKtbwv",
+            "src": "/images/favicon/android-chrome-192x192.png?v=vMddPLeYWy",
             "sizes": "192x192",
             "type": "image/png"
         },
         {
-            "src": "/images/favicon/android-chrome-512x512.png?v=fi5jKKtbwv",
+            "src": "/images/favicon/android-chrome-512x512.png?v=vMddPLeYWy",
             "sizes": "512x512",
             "type": "image/png"
         }
     ],
     "theme_color": "#ffffff",
     "background_color": "#ffffff",
+    "start_url": "https://decred.org",
```

Comment:
I'm not too familiar with the manifest.json, but from what I read about `start_url`, it seems like it should be a relative URL. If it is absent, it is the new desktop icon supposedly uses the location from which it was created.  Another common recommendation (MDN and developers.google.com) seems to be  `"start_url": "."`

Reply:
good spot. Fixed

Reply:
I will try to test this out on a phone, but it's still a little unclear what `"."` means.  From https://developer.mozilla.org/en-US/docs/Web/Manifest:

> The URL that loads when a user launches the application (e.g. when added to home screen), typically the index. Note that this has to be a relative URL, *relative to the manifest url.*

Emphasis added by me.